### DISCUSSION TITLE
Test Double acquires Pathfinder Product

### DISCUSSION THREAD

**Discussion by cathycolliver** [therubyradar/submissions#186] [Submissions]
Link: https://blog.testdouble.com/posts/2023-11-08-test-double-acquires-pathfinder/

Why Ruby Radar audience would care: Amidst a lot of tumultuous news in tech, here's a positive story about a successful software development consultancy that's been around over a decade acquiring a successful early stage product management consultancy. And both agencies started out in the same city of Columbus, Ohio, but have helped clients ranging from startups to major national brands.

**Comment by cjilbert504** [comment]
Thanks for submitting, @cathycolliver!!!
